app/policies/application_policy.rb

```diff
@@ -8,34 +8,6 @@ def initialize(user, record)
     @record = record
   end
 
-  def index?
```

Comment:
I like your approach of deleting unused code to increase test coverage 😆

Afaik, those methods are default for application policy - in case you forget to define policy method in class that  inherits from ApplicationPolicy, it will fallback here and return `false`. Now, when you delete them, it will raise exception. Which isn't necessarily bad, but you need to be aware of that.